platform/config/imports/spec.json

```diff
@@ -81,6 +81,15 @@
 		"toc": true,
 		"formats": ["websites","email","stories", "ads"]
 	},
+	{
+		"title": "Video in AMP",
+		"order": -1,
```

Comment:
I'm not sure what to do with this property. 🙂

Reply:
It defines the order of the pages in the sidebar: 
![image](https://user-images.githubusercontent.com/12857772/110483368-f28d1680-80e9-11eb-9a07-e896f2b00935.png)

See the order attributes of the other docs in [this collection ](https://github.com/ampproject/amp.dev/tree/future/pages/content/amp-dev/documentation/guides-and-tutorials/learn/spec)or pick something extremely high to add it to the end of the list.